packages/pn-pa-webapp/src/components/NewNotification/Recipient.tsx

```diff
@@ -464,33 +464,6 @@ const Recipient: React.FC<Props> = ({
                       />
                     </Grid>
                   </Grid>
-
-                  {paymentMode !== PaymentModel.NOTHING && (
```

Comment:
Remove creditorTaxId and noticeCode also from formik. Check this variables into the file